blog/src/components/Featured/Featured.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import CardWide from './CardWide'
+import { blogs } from "../../data/blog-posts";
+
+export default function Featured() {
+  return (
+    <div role='featured' className="container">
+        <section className="featured-sc bg-white">
+          <CardWide blogItemData = { blogs[0]} />
+        </section>
+      </div>
+  )
+}
```

Comment:
I think it would be better to pass the blogs list to the Featured component as Prop. Then loop over all the blogs to show card-wide for each blog.